asset-transfer-basic/chaincode-javascript/package-lock.json

```diff
@@ -0,0 +1,3246 @@
+{
```

Comment:
I've always assumed package-lock.json would be generated when user does 'npm install', rather than checking it in. Is there any advantage to checking it in?

Reply:
done. removed it from commit, and added it to .gitignore